lib/util.js

```diff
@@ -197,6 +198,18 @@ function pad(n) {
   return StringPrototypePadStart(n.toString(), 2, '0');
 }
 
+/**
+ * @param {string} format
+ * @param {string} text
+ * @returns {string}
+ */
+function styleText(format, text) {
+  validateOneOf(format, 'format', ObjectKeys(inspect.colors));
```

Comment:
```suggestion

/** @type {Set<string>} */
let colorsSet;
/**
 * @param {string} format
 * @param {string} text
 * @returns {string}
 */
function styleText(format, text) {
  colorsSet ??= new Set(ObjectKeys(inspect.colors));
  validateOneOfSet(format, 'format', colorsSet);
```

And we would need a version of `validateOneOf` that accepts a set rather than an array.

Reply:
I highly doubt that using a `Set` would be more performant. That being said it might be worth to call `ObjectKeys` only in the error path:
```suggestion

/**
 * @param {string} format
 * @param {string} text
 * @returns {string}
 */
function styleText(format, text) {
  validateString(format, 'format');
  const formatCodes = inspect.colors[format];
  if (formatCodes == null) {
    validateOneOf(format, 'format', ObjectKeys(inspect.colors));
  }
```

Reply:
We could at least call `ObjectKeys` only once, like `formatCodes ??= ObjectKeys(inspect.colors)`